expansion/src/main/java/no/unit/nva/expansion/model/ExpandedImportCandidate.java

```diff
@@ -44,7 +44,7 @@
 @JsonTypeName(ExpandedImportCandidate.TYPE)
 public class ExpandedImportCandidate implements ExpandedDataEntry {
 
-    public static final String TYPE = "ImportCandidate";
+    public static final String TYPE = "ImportCandidateSummary";
```

Comment:
Kjetil ønsker å ha det navnet for ImportCandidate fra indeksen.

Reply:
Den er grei